@here/harp-materials/lib/ShaderChunks/ExtrusionChunks.ts

```diff
@@ -6,14 +6,46 @@
 
 export default {
     extrusion_pars_vertex: `
-attribute vec3 extrusionAxis;
+attribute vec4 extrusionAxis;
```

Comment:
Please add documentation what is in this vec4 (especially in the w component)

Reply:
Done.